public/editor/scripts/main.js

```diff
@@ -38,6 +38,19 @@ require(["jquery", "bowser"], function($, bowser) {
     });
   }
 
+  Bramble.once("updatesAvailable", function() {
```

Comment:
Move all this new code down below line 65 where we check to make sure `window.Bramble` exists (it might not if we can't load `bramble.js`